CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 The changelog for [Kommunicate-iOS-SDK](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK). Also see the [releases](https://github.com/Kommunicate-io/Kommunicate-iOS-SDK/releases) on Github.
 
 ## [Unreleased]
+- [CM-956] UITestCases Optimization & bitrise updation 
```

Comment:
Here it is fine, but don't mention this in the release notes. This is not a customer facing issue.

Reply:
@mukeshthawani got it.